doc/fastfetch.1.in

```diff
@@ -0,0 +1,146 @@
+.TH FASTFETCH 1 "@FASTFETCH_BUILD_DATE@" "@CMAKE_PROJECT_NAME@ @CMAKE_PROJECT_VERSION@"
+
+.SH NAME
+fastfetch - a neofetch-like tool for fetching system information and
+displaying them in a pretty way
+
+.SH SYNOPSIS
+
+.B fastfetch
+.I [options...]
+
+.SH DESCRIPTION
+
+Fastfetch is a neofetch-like tool for fetching system information and
+displaying them in a pretty way. It is written mainly in C, with performance
+and customizability in mind.
+Currently, Linux, Android, FreeBSD, MacOS and Windows 7+ are supported.
+
+.SH "EXIT STATUS"
+
+On successfull execution, fastfetch returns zero. If any error happened,
+the exit code will be non-zero.
+
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
+Show help output, displaying all available options or help for given command
+.TP
+
+.B \-\-version
```

Comment:
We have `-v` as a shortcut

Reply:
fixed in 5445ef1581ab2cc8b5366fa49696225f555e3a61